addon/components/paper/text-field.js

```diff
@@ -0,0 +1,62 @@
+import Component        from 'ember-component';
+import formFieldOptions from
+  'ember-railio-form-components/mixins/form-field-options';
+import { computed, get, set } from '@ember/object';
+
+export default Component.extend(formFieldOptions, {
+  lazy: false,
```

Comment:
Why do you want to get rid of that structure?

Reply:
Because I think is overcomplicated. When something goes wrong in the number-field (for example), the problem could be in four different files. If you don't know the addon it's hard see how things work and why. Getting rid of the 'extend from extend from extend' should make it easier to oversee things.